### ISSUE TITLE
Upgrade from Chromium 106.0.5249.40 to Chromium 106.0.5249.55

### ISSUE THREAD

**Issue by emerick** [brave/brave-browser#25531]
Minor Chromium bump.

https://chromium.googlesource.com/chromium/src/+log/106.0.5249.40..106.0.5249.55?pretty=fuller&n=10000

QA tests:

Check branding items
Check for version bump

Additional checks:

No specific code changes in Brave (only line number changes in patches).

**Comment by kjozwiak** [comment]
The above requires `1.44.97` or higher for `1.44.x` verification 👍

**Comment by kjozwiak** [comment]
We'll still need to run through the affected area's mentioned via https://github.com/brave/brave-browser/issues/24517#issue-1331902953 on `macOS`, `Linux` and `Android Tablet` as they haven't ran through the affected areas.

Both `Windows` & `Android ARM (mobile)` just need to run a few spot checks.

**Comment by kjozwiak** [comment]
Verification **PASSED** on `Pixel 6` running `Android 13` using the following build(s):

```
Brave | 1.44.97 Chromium: 106.0.5249.55 (Official Build) (64-bit)
--- | ---
Revision | 4d5f098fca6ab7f4b6b7c240be3d9593c2357709-refs/branch-heads/5249@{#531}
OS | Android 13; Build/T1B1.220819.006
```

**`Example`** | **`Example`**
----------------|------------------
![Screenshot_20220922-151336](https://user-images.githubusercontent.com/2602313/191845258-561f8637-8e0c-45c8-8cc2-0e7b8f3a3691.png) | ![Screenshot_20220922-151342](https://user-images.githubusercontent.com/2602313/191845265-f4227de6-3137-414c-a925-502598624f45.png)

**Comment by GeetaSarvadnya** [comment]
Verification PASSED on

```
Brave | 1.44.97 Chromium: 106.0.5249.55 (Official Build) (64-bit)
-- | --
Revision | 4d5f098fca6ab7f4b6b7c240be3d9593c2357709-refs/branch-heads/5249@{#531}
OS | Windows 10 Version 21H2 (Build 19044.2006)

```
![image](https://user-images.githubusercontent.com/38657976/191941312-e6c0fd92-8dbc-43dc-802d-f220f6112420.png)

**Comment by stephendonner** [comment]
## Verification `IN-PROGRESS` 


Brave | 1.44.97 Chromium: 106.0.5249.55 (Official Build) (x86_64)
-- | --
Revision | 4d5f098fca6ab7f4b6b7c240be3d9593c2357709-refs/branch-heads/5249@{#531}
OS | macOS Version 11.7 (Build 20G817)


<details>
<summary>Color scheme (omnibox)</summary>

- Verified that omnibox color scheme is shown as expected in focused/not focused in light theme
- Verified that omnibox color scheme is shown as expected in focused/not focused in dark theme

**_Light theme_**

focused | not focused
-- | --



**_Dark theme_**

focused | not focused
-- | --


</details>

<details>
<summary>History page</summary>

- Confirmed that history of all the visited websites is listed in brave://history
- Confirmed that brave://history UI looks expected
- Confirmed that user can search the specific brave history via Search option
- Confirmed that imported browser history is shown in brave is as expected

Example | Example | Example | Example
-- | -- | -- | --


</details>

<details>
<summary>Content settings</summary>

- Confirmed content settings are working as expected

**_1. Block cookies content settings_**

- Open a site cnn.com
- Open content settings for cnn.com
- Block content settings for `Location` and `Window management`
- Reload cnn.com
- Ensured that `Location` and `Window management` permission are blocked for cnn.com
- Open brave://settings/cookies and `Block all cookies` 
- Reload cnn.com and ensured that all cookies are blocked on cnn.com

Example | Example | Example | Example | Example
-- | -- | -- | -- | --


**_2. Block scripts content settings_**
- Block the java scripts for a site via brave://settings/content/javascript
- Open the site ensured Java scripts blocked for the site
- Open a bunch of websited and ensured java script is not blocked

Example | Example | Example
-- | -- | --



**_3. Block autoplay global settings_**

- Block the autoplay content global settings via brave://settings/content/autoplay
- Open youtube.com and ensured no videos are autoplayed 
- Click on a video and play the video manully, wait till the 1st video ends
- Ensured 2nd video is not autoplayed 

</details>

<details>
<summary>Bookmarks</summary>

- Confirmed user can add/edit/delete the bookmarks
- Confirmed bookmarks can be imported via `NTP`, `brave://settings/importData`, `Hamburger menu - import bookmarks` option and `brave://bookmarks/ - Import bookmarks`
- Confirmed imported bookmarks can be opened in PT, new window, new tab and in a regular window
- Confirmed that imported bookmarks are shown in Private and Tor window new tabs

Example | Example | Example
-- | -- | --

 
</details>

<details>
<summary>Settings UI</summary>

- Verified that click on each settings in brave://settings redirected to the intended page 
- Verified the brave://settings UI and ensured UI looks good
- Verified the icons in the brave://settings page and ensured icons are shown as expected in each settings page
- Additional testing on settings is covered in "Settings comparision" section
</details>

<details>
<summary>Cookies</summary>

- Ensured that cookies are retained in an upgraded profile
- Ensured cookies settings are working as expected
- Ensured local and global cookies shield settings are working as expected

**Case 1:**
- Open a site cnn.com
- Open content settings for cnn.com
- Open brave://settings/cookies and `Block all cookies` 
- Reload cnn.com and ensured that all cookies are blocked on cnn.com
- Open brave://settings/cookies and `Allow all cookies` 
- Reload cnn.com and ensured that all cookies are allowed on cnn.com

Example | Example
-- | --


</details>

<details>
<summary>Multiple profiles</summary>

- Confirmed that user can create the multiple profiles (e.g `Profile 1` and `Profile 2`)
- Change the brave://settings and import bookmarks and add extensions in `Profile 1`
- Ensured that changes made in settings in Profile 1 are NOT reflected in Profile 2
- Ensured that the bookmarks and extensions are NOT reflected in Profile 2
- Enable rewards and update the rewards settings in brave://rewards in Profile 1
- Ensured that updated rewards settings are specific to profile 1 and are not reflected in Profile 2

</details>

<details>
<summary>First run dialog</summary>

- Confirmed first run dialog is shown when brave is launched in a clean install 
- Confirmed that the dialog content is as expected
- Confirmed click on `Set Brave as default` button in first-run dialog set the brave as the default browser and `Make Default` is removed from the brave://settings/getStarted
- Confirmed click on the `Maybe later` button in the first-run dialog doesn't set the brave as the default browser and the `Make Default` option is shown in the brave://settings/getStarted 
- Confirmed click on `x`  close button in first-run dialog and doesn't set the brave as the default browser and `Make Default` is shown in the brave://settings/getStarted
- Confirmed that the user can set the brave as default browser by clicking on the `Make Default`  option is shown in the brave://settings/getStarted 


</details>

<details>
<summary>Side panel</summary>

- Confirmed that sidebar is shown by default in the nightly channel
- Confirmed that `Reading list` icon is added to the sidebar menu
- Confirmed that reading list panel is shown when user click on the `Reading list` icon in the sidebar menu
- Confirmed that user can add the `list of websites` to the reading list by clicking on the `+Add current tab`
- Confirmed that user can add the `list of websites` to the reading list by right clicking on the tab and choose the option `Add tab to the reading list`
- Confirmed that `reading list panel can be hidden` by clicking on the `Reading list` icon in the sidebar menu
- Confirmed that reading list icon is shown in the URL bar, next to wallet icon
- Confirmed that user can enable/disable the reading list by clicking on the reading list icon (shown in the URL bar)
- Confirmed that reading list is shown in all the window types (Normal, PT, TOR and Guest window)
- Websites can't be added to reading list in Guest, encountered https://github.com/brave/brave-browser/issues/24930
- Confirmed that sites can be segregated to `Read` and `Unread` list
- Confirmed that sites can be deleted from the reading list

Example | Example 
-- | -- 
 


</details>

<details>
<summary>New tab page</summary>

- Ensured Tops sites can be edit/delete via NTP
- Ensured that NTP stats can be hidden/shown
- Ensured that a site can be added via NTP 
- Ensured NTP Widget cards can be added/removed
- Ensured NTP SI works as expected
- Ensured that click on settings (bookmarks, history, settings and brave talk) shortcuts works as expected
- Ensured that the existing features `upload your own` and `Brave background` works as expected
- Confirmed that `NTP SI` are shown in the order 2, 4, 8 tabs
- Confirmed that user can select the solid background for NTP
- Confirmed that user can choose the individual solid background via NTP
- Confirmed that same solid background is shown on each the NTP when the individual solid background is selected
- Confirmed that the different solid background is shown on each the NTP when `Refresh on every new tab` switch is ON
- Confirmed that user can select the gradient background for NTP
- Confirmed that user can choose the individual gradient background via NTP
- Confirmed that same gradient background is shown on each the NTP when the individual gradient background is selected
- Confirmed that the different gradient background is shown on each the NTP when `Refresh on every new tab` switch is ON
- Encountered #25392, #25393 and #25398

</details>

----------------

**_Settings comparison_**

<details>

Verified that the Language settings are updated as part of CR 106
![image](https://user-images.githubusercontent.com/38657976/191171416-6b9e1f01-ad6f-44ae-9947-4c570e0b9e4d.png)
Verified that content additional setting is renamed as `Windows management` as part of CR 106
![image](https://user-images.githubusercontent.com/38657976/191171737-999e5828-835b-4209-b702-9cd85c1e1d47.png)
New settigs `Tab hover mode` and `Side panel` settings are added as part of Brave in nightly (1.45.x) which are already merged in Beta (1.44.x)
![image](https://user-images.githubusercontent.com/38657976/191173573-d974423c-3fd9-4500-9b9c-4dff10f287ec.png)
Verified that `Reading list` sync data is been added in brave://sync as part of brave
![image](https://user-images.githubusercontent.com/38657976/191189679-c58b2daa-3e17-4a3b-ba1b-de30c3e9d075.png)
Verified the each settings in brave://settings page and ensured each settings works as expected

</details>

**_Components updater_**

<details>

- No new components are added as part of CR 106
- Components are updates successfully without any error
- Speedreader component is removed from nightly which is expected as per https://github.com/brave/brave-browser/issues/25116


**`1.43.93 Chromium: 105.0.5195.127`** | **`1.45.54 Chromium: 106.0.5249.40`**
-- | --


</details>

**_Brave startup calls_**
- Confirmed that only brave domains are listed in the Charles when the nightly build is launched with a clean profile
- Confirmed no calls to google, gstatic, etc domains when the nightly build is launched with a clean profile
